products/views.py

```diff
@@ -52,71 +52,63 @@ def get(self, request):
-                        "related_product_id"        : relate_product.related_product_group.id,
-                        "related_product_name"      : relate_product.related_product_group.name,
-                        "related_product_price"     : relate_product.related_product_group.price,
-                        "related_product_thumbnail" : relate_product.related_product_group.thumbnail
-                    } for relate_product in product_group.relate.all().select_related("related_product_group")],
-
-                    "question" :[{
-                        "product_group_id" : question.product_group.id,
-                        "question_id"      : question.id,
-                        "user_name"        : question.user.name,
-                        "title"            : question.title,
-                    } for question in product_group.question_set.all()]
-                }
-                cache.set(f"product_group_{product_group_id}", context)
+            product_group    = ProductGroup.objects.get(id = product_group_id)
+            product_packages = product_group.productgrouppackagetype_set.all()
+            context          = {}
+            context["result"] = {
+                "product_group_id"   : product_group.id,
+                "product_group_name" : product_group.name,
+                "sales_unit"         : product_group.sales_unit,
+                "thumbnail"          : product_group.thumbnail,
+                "delivery_type"      : product_group.delivery_type,
+                "information"        : product_group.information,
+                "main_description"   : product_group.main_description,
+                "detail_description" : product_group.detail_description,
+                "discount_rate"      : product_group.discount_rate,
+
+                "package_type" : product_packages[0].package_type.name,
+
+                "product_group_images" : [{
+                    "url" : image.url
+                } for image in product_group.productgroupimage_set.all()],
+
+                "products" : [{
+                    "id"         : product.id,
+                    "name"       : product.name,
+                    "price"      : product.price,
+                    "sold_out"   : product.sold_out,
+                    "ingredient" : product.ingredient,
+                    "point"      : product.point
+                } for product in product_group.product_set.all()],
+
+                "related_product" : [{
+                    "related_product_id"        : relate_product.related_product_group.id,
+                    "related_product_name"      : relate_product.related_product_group.name,
+                    "related_product_price"     : relate_product.related_product_group.price,
+                    "related_product_thumbnail" : relate_product.related_product_group.thumbnail
+                } for relate_product in product_group.relate.all().select_related("related_product_group")],
```

Comment:
```suggestion
                } for relate_product in product_group.relate.select_related("related_product_group")],
```
꼭 수정하지는 않으셔도 됩니다.
`all()` 없이도 동작합니다